src/bch/coin-selection.js

```diff
@@ -73,9 +73,9 @@ const selectAll = (feePerByte, coins, outAddress) => {
 
 // descentDraw :: [Coin] -> Number -> [Coin] -> String -> Selection
 const descentDraw = (targets, feePerByte, coins, changeAddress) => {
-  let splitCoins = prepareForSplit(coins)
+  let splitCoins = sort(Coin.descentSort, coins); // prepareForSplit(coins)
```

Comment:
Same as above, would be nice if `prepareForSplit` weren't build into this function.